### DISCUSSION TITLE
Working with Chainlink price feeds

### DISCUSSION THREAD

**Discussion by yeahChibyke** [Cyfrin/foundry-full-course-cu#655] [Q&A]
When working with a ETH/USD price feed, we multiply the 'price' variable by 1e10.
When working with a BTC/USD price feed, do I multiply the 'price' variable by 1e10 also? Especially since BTC has a decimal place of 1e8 unlike ETH which has 1e18.
I have asked Phind, but I am not clarified with the answer it gave me.

**Comment by shikhar229169** [comment]
Check this out:
https://github.com/Cyfrin/2023-07-foundry-defi-stablecoin/issues/1028
